tests/metagen/typegraphs/sample/ts/client.ts

```
@@ -1072,10 +1072,12 @@ export type RootGetUserFnInput = Record<string, never>;
 export type UserEmailStringEmail = string;
 export type UserPostsPostList = Array<Post>;
 export type User = {
-  id: UserIdStringUuid;
-  email: UserEmailStringEmail;
+  id: StringUuid4;
+  email: StringEmail5;
   posts: UserPostsPostList;
 };
+export type StringEmail5 = string;
+export type UserPostsPostList = Array<Post>;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UserPostsPostList'. Consider to delete it or rename it.

'UserPostsPostList' is defined here:


(lint/suspicious/noRedeclare)